mantidproject/mantid
# Make Engineering Diffraction Calibration fit Quadratically

**Description of work.**
The original implementation made no use of the 'DIFA' value when fitting. This PR adds that functionality and integrates it into both Engineering Diffraction GUIs. 

**To test:**
(Test in both GUIs, Engineering Diffraction 1 only available in MantidPlot)
1. Open the engineering Diffraction GUI
2. Run a calibration (V: 307521, C: 305738)
3. Check the `engggui_calibration_banks_parameters` workspace has values for `DIFA`
3. Check that the generated file contains `INS  2 ICONS  18373.00    5.73    2.42 ` or similar where the middle value is not zero. The values should match the ones in the `engggui_calibration_banks_parameters` table workspace. 

Fixes #27635 . 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
